### PR TITLE
add news and events content

### PR DESCRIPTION
**Pull request by myrcutio**

### CODE REVIEW

File: app/src/Routes/NewsEvents/NewsEvents.js [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+
+const NewsEvent = (event) => (
+    <figure>
+        <figcaption>{event.title}</figcaption>
+        <figcaption>{event.subtitle}</figcaption>
+        {event.location ? (<span>in {event.location}</span>): ""}
+        {event.date ? (<span> {event.date}</span>): ""}
+        <p>{event.description}</p>
+        {event.postscript ? (<p>{event.postscript}</p>) : ""}
+    </figure>
```
Comment (steppy452) [inline app/src/Routes/NewsEvents/NewsEvents.js:13]: same comment as w/ the works page - this isn't the right use for `figure` and `figcaption` elements

File: app/src/Routes/NewsEvents/NewsEvents.js [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+)
 
 export const NewsEvents = () => (
   <div>
     <Typography variant="h6">NewsEvents</Typography>
-    <p></p>
+      {events.events ? events.events.map(e => NewsEvent(e)) : ""}
```
Comment (steppy452) [inline app/src/Routes/NewsEvents/NewsEvents.js:19]: the `.map` needs to set a `key` somewhere for react to detect changes

File: app/src/Routes/NewsEvents/NewsEvents.js (hunk 1 above)
Comment (myrcutio) [reply app/src/Routes/NewsEvents/NewsEvents.js]: sounds good, fixed

File: app/src/Routes/NewsEvents/NewsEvents.js (hunk 2 above)
Comment (myrcutio) [reply app/src/Routes/NewsEvents/NewsEvents.js]: should be good now

**Pull request merged**